### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Not 100% sure about the "E-mail forwarding address" strings.

### CODE REVIEW

File: portal/locales/en.json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-   "password_too_simple_2": "Password needs to be at least 8 characters long and contains digit, upper and lower characters",
-   "password_too_simple_3": "Password needs to be at least 8 characters long and contains digit, upper, lower and special characters",
-   "password_too_simple_4": "Password needs to be at least 12 characters long and contains digit, upper, lower and special characters",
-   "good_practices_about_user_password": "You are now about to define a new user password. The password should be at least 8 characters - though it is good practice to use longer password (i.e. a passphrase) and/or to use various kind of characters (uppercase, lowercase, digits and special characters).",
-   "wrong_current_password": "Current password is wrong",
-   "invalid_mail": "Invalid mail address",
+   "login": "Log in",
+   "logout": "Log out",
+   "password_changed": "Password changed",
+   "password_changed_error": "Could not change passowrd",
```
Comment (decentral1se) [inline portal/locales/en.json]:
```suggestion
   "password_changed_error": "Could not change password",
```

File: portal/locales/en.json [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+   "wrong_current_password": "The current password is wrong",
+   "invalid_mail": "Invalid e-mail address",
    "invalid_domain": "Invalid domain in",
-   "invalid_mailforward": "Invalid mail forward address",
-   "mail_already_used": "Mail address already used:",
-   "information_updated": "Information updated",
-   "user_saving_fail": "An error occurred on user's modification saving",
-   "missing_required_fields": "Required fields are missing",
+   "invalid_mailforward": "Invalid e-mail forwarding address",
+   "mail_already_used": "E-mail address already in use:",
```
Comment (decentral1se) [inline portal/locales/en.json]:
```suggestion
   "mail_already_used": "E-mail address already in use",
```

?

**Review by decentral1se — COMMENTED** [review]
:fire:

**Review by decentral1se — APPROVED** [review]

File: portal/locales/en.json [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-   "invalid_mailforward": "Invalid mail forward address",
-   "mail_already_used": "Mail address already used:",
-   "information_updated": "Information updated",
-   "user_saving_fail": "An error occurred on user's modification saving",
-   "missing_required_fields": "Required fields are missing",
+   "invalid_mailforward": "Invalid e-mail forwarding address",
+   "mail_already_used": "E-mail address already in use",
+   "information_updated": "Info updated",
+   "user_saving_fail": "Could not save new user info",
+   "missing_required_fields": "Fill inn the required fields",
```
Comment (alexAubin) [inline portal/locales/en.json]:
```suggestion
   "missing_required_fields": "Fill in the required fields",
```

**Review by alexAubin — APPROVED** [review]
Small typo, LGTM :+1:

File: portal/locales/en.json (hunk 3 above)
Comment (comradekingu) [reply portal/locales/en.json]: Nice catch

**Pull request merged**